WooCommerce/Classes/ViewModels/Order Details/OrderPaymentDetailsViewModel.swift

```diff
@@ -111,6 +109,19 @@ final class OrderPaymentDetailsViewModel {
         return String.localizedStringWithFormat(template, styleDate, order.paymentMethodTitle)
     }
 
+    /// Awaiting payment
+    ///
+    var awaitingPayment: String? {
+        if order.paymentMethodTitle.isEmpty {
+            return String.localizedStringWithFormat(
+                NSLocalizedString("Awaiting payment", comment: "Awaiting payment"))
```

Comment:
Nit: we should clarify the comments on `NSLocalizedString` to help translators understand the context better.
```suggestion
                NSLocalizedString("Awaiting payment", comment: "The title on the payment row of the Order Details screen when the payment is still pending"))
```